src/components/input/template.test.js

```diff
@@ -0,0 +1,113 @@
+/* globals describe, it, expect */
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const examples = getExamples('input')
+
+describe('Input', () => {
+  describe('by default', () => {
+    it('renders with classes', () => {
+      const { $ } = render('input', {
+        classes: 'app-c-input--custom-modifier'
+      })
+
+      const $component = $('.govuk-c-input')
+      expect($component.hasClass('app-c-input--custom-modifier')).toBeTruthy()
+    })
+
+    it('renders with id', () => {
+      const { $ } = render('input', {
+        id: 'my-input'
+      })
+
+      const $component = $('.govuk-c-input')
+      expect($component.attr('id')).toEqual('my-input')
+    })
+
+    it('renders with name', () => {
+      const { $ } = render('input', {
+        name: 'my-input-name'
+      })
+
+      const $component = $('.govuk-c-input')
+      expect($component.attr('name')).toEqual('my-input-name')
+    })
+
+    it('renders with type="text"', () => {
+      const { $ } = render('input')
+
+      const $component = $('.govuk-c-input')
+      expect($component.attr('type')).toEqual('text')
+    })
+
+    it('renders with value', () => {
+      const { $ } = render('input', {
+        value: 'QQ 12 34 56 C'
+      })
+
+      const $component = $('.govuk-c-input')
+      expect($component.val()).toEqual('QQ 12 34 56 C')
+    })
+
+    it('renders with attributes', () => {
+      const { $ } = render('input', {
+        attributes: {
+          'aria-describedby': 'my-label'
```

Comment:
It might be worth using a different 'non-sensical' data attribute, only to avoid confusion for people scanning through who might mis-read this as inputs actually having or requiring some aria markup?

Reply:
replaced `aria-describedby` with 'non-sensical' data attribute